samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/package.json

```diff
@@ -12,10 +12,10 @@
     "@testing-library/user-event": "^13.5.0",
```

Comment:
Why did you removed package-lock.json ??

Reply:
Restored